packages/tasks/src/model-libraries.ts

```diff
@@ -478,6 +478,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path:"MeshAnything_350m.pth"`,
 		snippets: snippets.mesh_anything,
 	},
+	"merlin": {
+		prettyLabel: "Merlin",
+		repoName: "merlin",
+		repoUrl: "https://github.com/StanfordMIMI/Merlin",
+		filter: false,
+		countDownloads: `path_filename:"i3_resnet_clinical_longformer_best_clip_04-02-2024_23-21-36_epoch_99.pt"`,
```

Comment:
```suggestion
		countDownloads: `path_extension:"pt"`,
```

I fear that putting the filename will be too specific. Better to count downloads on .pt files instead. This way if you update the model in the repo or create a new repo with another trained Merlin model, then we won't have to update the `countDownloads` again.